src/remove-dulplicates/index.js

```diff
@@ -1,3 +1,117 @@
-function removeDuplicates(obj) {}
+function removeDuplicates(obj) {
+
+
+    /* Steps to removing duplicates
```

Comment:
This isn't exactly necessary. If you think it is, should be outside the function. Look into `jsdoc` too while you're at it.